### DISCUSSION TITLE
　torch.sizeの不一致

### DISCUSSION THREAD

**Discussion by RRR-troisR** [espnet/espnet#3936] [Q&A] [answered]
vitsのfinetuningを行った際下記エラーが発生しました。解決策がわからなかったので質問させてください。

ダウンロードしたモデル
espnet_model_zoo_download --unpack true --cachedir downloads kan-bayashi/jsut_full_band_vits_accent_with_pause

コード
./run.sh \
    --stage 6 \
    --g2p pyopenjtalk_accent_with_pause \
    --min_wav_duration 0.19 \
    --fs 44100 \
    --n_fft 2048 \
    --n_shift 512 \
    --dumpdir dump/44k \
    --win_length null \
    --tts_task gan_tts \
    --feats_extract linear_spectrogram \
    --feats_normalize none \
    --train_config ./conf/tuning/finetune_full_band_vits.yaml \
    --train_args "--init_param downloads/2e3f4056cfe94ff3c015dc11bc0a6314/exp/tts_train_full_band_vits_raw_phn_jaconv_pyopenjtalk_accent_with_pause/train.total_count.ave_10best.pth:tts:tts" \
    --tag finetune_full_band_vits_raw_phn_jaconv_pyopenjtalk_accent_with_pause

エラー
RuntimeError: Error(s) in loading state_dict for VITS:
	size mismatch for generator.text_encoder.emb.weight: copying a param with shape torch.Size([85, 192]) from checkpoint, the shape in current model is torch.Size([69, 192]).

**Comment by kan-bayashi** [comment, marked as answer]
https://github.com/espnet/espnet/tree/master/egs2/jvs/tts1#3-replace-token-list-with-pretrained-models-one-1
忘れていませんか？

**Reply by RRR-troisR** [reply]
解決しました。ありがとうございます。重ねてもう一点質問したいのですがmodelzooの方で.aveや_latest とついているモデルとついてないモデルの差異はどのような感じなのでしょうか？

**Reply by kan-bayashi** [reply]
aveはモデルアベレージを行ったもので、latestは単純に最後のiterationのモデルですね。
